packages/lib/server/getLuckyUser.ts

```diff
@@ -92,7 +92,10 @@ async function leastRecentlyBookedUser<T extends Pick<User, "id" | "email">>({
   }
 
   const leastRecentlyBookedUser = availableUsers.sort((a, b) => {
-    return userIdAndAtCreatedPair[a.id] > userIdAndAtCreatedPair[b.id] ? 1 : -1;
+    if (userIdAndAtCreatedPair[a.id] > userIdAndAtCreatedPair[b.id]) return 1;
+    else if (userIdAndAtCreatedPair[a.id] < userIdAndAtCreatedPair[b.id]) return 0;
```

Comment:
It doesn't sort 3,2,1 to 1,2,3. That seems like an unintended behaviour change. Can you confirm @emrysal

Reply:
yeah it should be 

```suggestion
    else if (userIdAndAtCreatedPair[a.id] < userIdAndAtCreatedPair[b.id]) return -1;
```

Reply:
or we should probably use days() to compare dates

Reply:
comparing dates works fine as they are numeric in this comparison, good spot on the 0! It should indeed be -1